mobile_app_connector/models/__init__.py

```diff
@@ -10,4 +10,5 @@
 ##############################################################################
 
 from . import compassion_child
+from . import compassion_login
```

Comment:
I'm sorry I didn't see this before but it's better to name the file `res_users`. The convention is to name the file similar to the odoo object they represent, this helps finding relevant code.